src/config.ts

```diff
@@ -147,18 +147,19 @@ export class CacheConfig {
       for (const cargo_manifest of cargo_manifests) {
         try {
           const content = await fs_promises.readFile(cargo_manifest, { encoding: "utf8" });
-          const parsed = toml.parse(content);
+          // Use any since TomlPrimitive is not exposed
+          const parsed = toml.parse(content) as { [key: string]: any };
 
           if ("package" in parsed) {
             const pack = parsed.package;
-            if ("version" in pack) {
-              pack.version = "0.0.0";
+            if ("version" in Object.keys(pack)) {
```

Comment:
This is not right, `Object.keys()` returns an array, and `in` is checking for properties.
The previous `"version" in pack` should work just fine, otherwise you might want to use `Object.hasOwn(pack, "version")` if you want to be extra save.

Reply:
Thanks, I've reverted this change.